### ISSUE TITLE
Error to watching Kafka Topics

### ISSUE THREAD

**Issue by caiomedeirospinto** [kiegroup/kogito-operator#1090]
Hi community,

I'm here to report this error message that is continuos logged for the Kogito operator when it's installed on Openshift 4, this is the error message:

```log
(...)
E1213 12:39:59.382846       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.7/tools/cache/reflector.go:167: Failed to watch *v1beta2.KafkaTopic: failed to list *v1beta2.KafkaTopic: v1beta2.KafkaTopicList.Items: []v1beta2.KafkaTopic: v1beta2.KafkaTopic.Spec: v1beta2.KafkaTopicSpec.Config: ReadString: expects " or n, but found 6, error found in #10 byte of ...|tion.ms":604800000,"|..., bigger context ...|-e5bf91089b62"},"spec":{"config":{"retention.ms":604800000,"segment.bytes":1073741824},"partitions":|...
E1213 12:40:41.735221       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.7/tools/cache/reflector.go:167: Failed to watch *v1beta2.KafkaTopic: failed to list *v1beta2.KafkaTopic: v1beta2.KafkaTopicList.Items: []v1beta2.KafkaTopic: v1beta2.KafkaTopic.Spec: v1beta2.KafkaTopicSpec.Config: ReadString: expects " or n, but found 6, error found in #10 byte of ...|tion.ms":604800000,"|..., bigger context ...|-e5bf91089b62"},"spec":{"config":{"retention.ms":604800000,"segment.bytes":1073741824},"partitions":|...
(...)
```

Full log:

[kogito-operator-controller-manager-84f9d8c74b-sbg76-manager.log](https://github.com/kiegroup/kogito-operator/files/7704859/kogito-operator-controller-manager-84f9d8c74b-sbg76-manager.log)

It makes the operator lose focus on reconciliation, it doesn't interact in front of new CRs. Is there a workaround?

The list of versions used:

* Openshift 4.7.38
* Kubernetes v1.20.11+aa025a0
* Kogito Operator 1.13.0
* Infinispan Operator 2.2.1
* Strimzi 0.26.0

**Comment by sutaakar** [comment]
@caiomedeirospinto Thanks for reporting this. I have created a JIRA to address this issue - https://issues.redhat.com/browse/KOGITO-6414
A workaround would be to remove KafkaTopic with config section containing non string values, or use string based config values.

**Comment by sutaakar** [comment]
@caiomedeirospinto The operator fix should be available in version 1.15

**Comment by sutaakar** [comment]
Fixed in 1.15
